app/src/main/java/com/yacgroup/yacguide/TableActivity.kt

```diff
@@ -66,7 +67,7 @@ abstract class TableActivity : BaseNavigationActivity() {
             Comment(
                 text = it.text.orEmpty(),
                 properties = listOf(
-                    CommentProperty(R.id.qualityLayout, R.string.relevance, RegionComment.QUALITY_MAP, it.qualityId)
+                    CommentProperty(R.string.relevance, RegionComment.QUALITY_MAP, it.qualityId)
```

Comment:
- [ ] Please check e.g. in Zittauer Gebirge - Weißbachtal - Kleiner Grenzkogel -  Alter Weg. When showing the comments dialog, the comment's text is placed above the drying info, **which is a regression**. Another example is Zittauer Gebirge - Weißbachtal - Kleiner Grenzkogel -  Alter Weg. In the first comment, the comment's text is placed above protection, too.

Seems to be a flakiness since most comments look ok as far as I could see.

Reply:
Found the issue.